### PR TITLE
Update ctapipe_io_lst to 0.8.0

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

**Review by moralejo — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ are unit tested and should be working as long as the build status is passing.
 
 - You will need to install [anaconda](https://www.anaconda.com/distribution/#download-section) first. 
 
+
+### macOS requirements
+
+lstchain depends on `protozfits` to read LST raw data via `ctapipe_io_lst`.
+Currently, prebuilt binaries are only available on linux.
+On macOS, you will need `protobuf` and `zeromq` installed so that the
```
Comment (rlopezcoto) [inline README.md:20]: shouldn't we then include that in the environment? or at least an option for macos

Comment (maxnoe) [reply README.md:20]: I am working on making it work with `protobuf` and `zeromq` installed via conda and also on a conda-forge recipe, that should also work for mac. Should be done tomorrow I hope.

**Comment by moralejo** [comment]
Is this finished then?

**Review by moralejo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rlopezcoto) [reply README.md:20]: thanks

**Comment by maxnoe** [comment]
@moralejo  I think so yes. Currently working on the conda recipe, but ran into an issue there. Hope to get it out today still but the conda package will probably take a while since it has to be approved by the conda-forge guys.

**Comment by moralejo** [comment]
But we can merge and it will work on linux, right? And on Mac once the conda package is there.

**Comment by maxnoe** [comment]
@moralejo on mac should also already work, but it will compile from source, so installing might take 1-2 minutes.

**Comment by moralejo** [comment]
Can we merge then? I'd like to test it, and that makes it easier.

**Comment by maxnoe** [comment]
Fine with me

**Review by rlopezcoto — APPROVED** [review]

**Comment by maxnoe** [comment]
I tested in my macvm and everything works fine (runing the tests, haven't tried more).

If mac support is important, we could try activating mac builds in the github CI. It was just not good on travis, but github actions has mac support I think.

**Comment by moralejo** [comment]
> I tested in my macvm and everything works fine (runing the tests, haven't tried more).
> 
> If mac support is important, we could try activating mac builds in the github CI. It was just not good on travis, but github actions has mac support I think.

Would be good yes, many of us use mac. This will go in parallel and not slow the CI tests, right?

**Comment by maxnoe** [comment]
Yes, should go in parallel.

**Comment by rlopezcoto** [comment]
maybe not making it a `Required` test for the PRs to be accepted? Although I agree with @moralejo that many of us work with Mac and it would be pretty needed that lstchain works in this OS.

**Comment by Bultako** [comment]
just for info
I had to update `cmake` compiler to version>3.15.0 to build the conda env in my mac

**Comment by maxnoe** [comment]
Yes, that is required.

**Pull request merged**